src/dotplot.py

```diff
@@ -10,19 +10,19 @@
 
 mshape = input.shape
 
-print(mshape)
-print(input)
+print(f' > Input shape: {mshape}')
```

Comment:
Since this one is python 3.6 in advance, you will have to change the README to indicate the minimum python version required. Otherwise you can just do something less beautiful:
```
print("The input shape is")
print(mshape)
```

Reply:
In that case .format() could be used, which was introduced in Python 2.6, i'm working in that right now.

It would be `print('The input shape is {}'.format(mshape))`

Reply:
Sure, that would do it. Or you can add min Python 3.6 to the requirements, etc. Up to you.